src/materialized/src/http/prof.rs

```diff
@@ -11,10 +11,12 @@
 
 use std::future::Future;
 
+use askama::Template;
 use cfg_if::cfg_if;
 use hyper::{Body, Request, Response};
 
 use crate::http::Server;
+use prof::ProfStartTime;
```

Comment:
This looks to me like it is going to break on macOS because the `prof` crate is only imported on Linux: https://github.com/MaterializeInc/materialize/blob/8ea75aa76992a1a1c2a2580b8c5047736ad78d7a/src/materialized/Cargo.toml#L62-L71